### PR TITLE
created fb login

### PR DESCRIPTION
**Pull request by po4erk**
fb login for Android, need created settings for IOS with API and reducers

### CODE REVIEW

File: src/screens/SignIn/index.js [hunk 1]
```diff
@@ -56,14 +62,23 @@ class SignIn extends React.Component<Props, State> {
...
+
 	render() {
 		return (
 			<Container style={styles.container}>
 				<ScrollView>
-					<Button block style={styles.fbButton}>
-						<Icon name='logo-facebook' />
-						<Text> SIGN IN WITH FACEBOOK </Text>
-					</Button>
+          <FBLogin
```
Comment (iarickvigasi) [inline src/screens/SignIn/index.js]: Why wrong identation?

File: src/screens/SignIn/index.js [hunk 2]
```diff
@@ -56,14 +62,23 @@ class SignIn extends React.Component<Props, State> {
     }
   }
 
+  fbLogIn = (data) => {
+    this.setState({ fbUser : data.credentials })
+    if (this.state.fbUser) {
+      this.props.navigation.navigate('App')
```
Comment (iarickvigasi) [inline src/screens/SignIn/index.js]: Leave @TODO here please :)

File: src/screens/SignIn/index.js [hunk 3]
```diff
@@ -4,7 +4,9 @@ import { Container, View, Toast, Button, Icon, Text } from 'native-base'
 import t from 'tcomb-form-native'
 import { connect } from 'react-redux'
 import styles from './styles'
-import { signin } from '/services/session/actions'
+import { signin } from '../../services/session/actions'
+import { FBLogin, FBLoginManager } from 'react-native-facebook-login'
+import FBLoginView from '../../components/Facebook'
```
Comment (olbesp) [inline src/screens/SignIn/index.js]: Use absolute paths like '/components/...'

**Comment by iarickvigasi** [comment]
Conflicts

**Pull request merged**